lightning/src/onion_message/messenger.rs

```diff
@@ -283,6 +283,16 @@ where
 		&self, path: OnionMessagePath, message: OnionMessageContents<T>,
 		reply_path: Option<BlindedPath>
 	) -> Result<(), SendError> {
+
+		match &message {
+			OnionMessageContents::Offers(ref offers_msg) => {
```

Comment:
It'd be more idiomatic to implement `fmt::Display` for `OnionMessageContents`, that way we can keep the 1-line log